data/Makefile

```diff
@@ -1,28 +1,29 @@
 YEAR := 2017
 
 .PHONY: all clean
-all: final/acs/age_and_sex.csv \
-	 final/acs/household_income.csv \
-	 final/acs/employment_status.csv \
-	 final/acs/income_inequality.csv \
-	 final/acs/educational_attainment.csv \
-	 final/acs/means_of_transit.csv \
-	 final/acs/household_type.csv \
-	 final/acs/total_population.csv \
-	 final/acs/hours_worked_per_week.csv \
-	 final/acs/race.csv \
-	 final/acs/hispanic_or_latino_origin.csv \
-	 final/acs/household_tenure.csv \
-	 final/acs/vacancy_status.csv \
-	 final/acs/disability.csv
+all: final/acs/census_observations.csv
```

Comment:
Since an ACS count for a given FIPS code and variable is now stored as a `CensusObservation`, I updated the import script to merge all of the ACS data together. This is intended to follow our new best practices for data imports, where as much processing as possible is done in the Makefile and the `import_data` management command is scoped to simply loading the data into the database.